src/plugins/autonomy/ArduPilot/ArduPilot.cpp

```diff
@@ -180,7 +184,7 @@ bool ArduPilot::step_autonomy(double t, double dt) {
 
 void ArduPilot::handle_receive(const boost::system::error_code& error,
                                std::size_t num_bytes) {
-#if 0
+#if 1
```

Comment:
Should this #if evaluate to true on the "final" version of the ArduPilot plugin? Maybe wrap this in the enable_debug parameter. If you want different levels of debug, you could use a debug_level parameter instead, where debug_level is an integer.